crates/dbsp/src/trace/persistent/trace.rs

```diff
@@ -43,7 +43,12 @@ where
     approximate_len: usize,
 
     lower_key_bound: Option<B::Key>,
-    lower_val_bound: Option<B::Val>,
+
```

Comment:
so lower_key_bound above is still needed?

Reply:
`lower_key_bound` has a very efficient implementation (O(1) for the entire batch), so seems useful